### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SeyZ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,30 +6,16 @@
   <img src="https://github.com/ForestAdmin/lumber/blob/master/assets/lumber-logo.png?raw=true" alt="Lumber logo">
 </p>
 
-Lumber is a CLI tool that helps you create your backend application in minutes.
-At its core, Lumber has been designed with a modular architecture that leverages a set of plugins.
+Lumber is the CLI tool used to install [Forest Admin](https://www.forestadmin.com), the Admin Panel framework.
+Forest Admin has been designed with scalability in mind to fit requirements from small projects to mature companies.
 
-For example, it takes one command to get a REST API, a GraphQL API or a fully operational Admin Panel.
+## Getting started
```
Comment (marcperrinp) [inline README.md]: Maybe a bit too "austère".. How about: "Get your admin panel running in minutes by [installing Forest Admin](https://app.forestadmin.com/signup)."  ?

File: README.md [hunk 2]
```diff
@@ -6,30 +6,16 @@
...
 
-## Install
-
-```sh
-npm install -g lumber-cli -s
-```
+[https://app.forestadmin.com/signup](https://app.forestadmin.com/signup)
 
 ## See it in action
```
Comment (marcperrinp) [inline README.md]: Same here, maybe explaining in a few words what the live demo is can help:
"Check out our [Live demo](https://app.forestadmin.com/23065?livedemo) to see what Forest Admin looks like in practice."

**Comment by marcperrinp** [comment]
Also @SeyZ, does it make sense to keep the logo..? :/

**Comment by SeyZ** [comment]
> Also @SeyZ, does it make sense to keep the logo?

I agree that we should replace it by the Forest Admin's logo but in my opinion, we should merge "Lumber" and the "Forest CLI" project overall. So for now, I will keep it like this until we decide to work on the merge

**Pull request merged**